symforce/codegen/backends/rust/rust_config.py

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------
+# SymForce - Copyright 2022, Skydio, Inc.
+# This source code is under the Apache 2.0 license found in the LICENSE file.
+# ----------------------------------------------------------------------------
+from dataclasses import dataclass
+from pathlib import Path
+
+import sympy
+from sympy.printing.codeprinter import CodePrinter
+
+from symforce import typing as T
+from symforce.codegen.backends.rust import rust_code_printer
+from symforce.codegen.codegen_config import CodegenConfig
+
+CURRENT_DIR = Path(__file__).parent
+
+
+@dataclass
+class RustConfig(CodegenConfig):
+    """
+    Code generation config for the Rust backend.
+
+    Args:
+        doc_comment_line_prefix: Prefix applied to each line in a docstring
+        line_length: Maximum allowed line length in docstrings; used for formatting docstrings.
+        scalar_type: The scalar type to use (float or double)
+    """
+
+    doc_comment_line_prefix: str = "///"
+    line_length: int = 100
+    scalar_type: rust_code_printer.ScalarType = rust_code_printer.ScalarType.FLOAT
```

Comment:
I might suggest defaulting this to double - we have it default to float for CUDA because f32 performance there is often significantly better, but on CPU I generally default to double and that's what I would probably recommend people start with unless they know they need f32

Reply:
Done